### ISSUE TITLE
Mismatch between rspamd and sieve configuration

### ISSUE THREAD

**Issue by dymitar** [Intermesh/debian-groupoffice-mailserver-antispam#1]
Dear all,
I upgraded GO 6.2 to 6.3 according to the documentation and it is working perfectly except the sieve rule for Spam.
Current sieve configuration relies on "X-Spam-Flag" header to recognize spam, and the rspamd is using "X-Spam" by default. Initially I thought this is old configuration of GO 6.2, but I created new email account with 6.3 and it created the same file with "X-Spam-Flag". The problem is solved easy by changing the configuration of rspamd. You should put a file milter_headers.conf in /etc/rspamd/local.d/ with following content:

`    extended_spam_headers = true;
    spam-header{
        header = "X-Spam-Flag";
		value = "YES";
    }`
I don't know if this is good solution. May be better solution is to change the sieve functionality of GO to create proper configuration with "X-Spam" header. But this will require manual adjustment of all mail folders after upgrade.

One more remark - there is a file groupoffice.inc in /etc/rspamd/local.d/ , but I don't think this file is used. I don't see how this file is included in rspamd configuration.

**Comment by mschering** [comment]
Thanks for reporting! I renamed the header and config file name so it's included.

**Comment by dymitar** [comment]
I see you put it to worker_proxy configuration, but I'm not sure if this will work.
I tried changing worker_proxy and it didn't change the header name (may be I didn't do it correctly, i'm not very familiar with rspamd). After I did this in milter_headers it worked.
I tested this using https://www.emailsecuritycheck.net/
One more remark - I had issues with just reloading rspamd, I had to restart it after configuration change.
